examples/src/contracts/balance_checker.rs

```diff
@@ -1,16 +1,24 @@
 //! Example of Balance Checker contract.
-use odra::prelude::*;
 use odra::{casper_types::U256, Address};
+use odra::{prelude::*, External};
 
 /// BalanceChecker contract.
 #[odra::module]
-pub struct BalanceChecker;
+pub struct BalanceChecker {
+    token: External<TokenContractRef>
+}
 
 #[odra::module]
 impl BalanceChecker {
+    /// Initializes the contract with the given parameters.
+    pub fn init(&mut self, address: Address) {
+        self.token.set(address);
```

Comment:
What if ecternal contract has the `set()` method?

Reply:
If a contract implements the `set()` method, you can't use the deref [coercion mechanism](https://doc.rust-lang.org/std/ops/trait.Deref.html#deref-coercion), so you call the `External.set()` method. In this case you need to dereference explicitly:  
```rust
use core::ops::DerefMut;
use odra::{Address, External, prelude::*};

#[odra::module]
pub struct Contract {
    ext: odra::External<SetterContractRef>
}

#[odra::module]
impl Contract {
    pub fn init(&mut self, address: odra::Address) {
        self.ext.set(address);
    }

    pub fn set(&mut self, value: bool) {
        self.ext.deref_mut().set(false);
        // or
        DerefMut::deref_mut(&mut self.ext).set(false);
    }
}

#[odra::external_contract]
pub trait Setter {
    fn set(&mut self, value: bool);
}
```

Reply:
Please add this to the docs of `Extern`